### PR TITLE
[Attestation] Fix #3937 #4120: `az attestation policy set` and `az attestation policy show` fails when execution

### LINKED ISSUE #3937: Error when querying for current attestation policy
## Describe the bug

**Command Name**
`az attestation policy show
Extension Name: attestation. Version: 0.2.0.`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
It is required that you pass in a value for the "algorithms" argument when calling decode().
Traceback (most recent call last):
  File "/opt/az/lib/python3.6/site-packages/knack/cli.py", line 231, in invoke
    cmd_result = self.invocation.execute(args)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 657, in execute
    raise ex
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 720, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 691, in _run_job
    result = cmd_copy(params)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 328, in __call__
    return self.handler(*args, **kwargs)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/command_operation.py", line 363, in handler
    show_exception_handler(ex)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/arm.py", line 429, in show_exception_handler
    raise ex
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/command_operation.py", line 361, in handler
    return op(**command_args)
  File "/home/csegarra/.azure/cliextensions/attestation/azext_attestation/manual/custom.py", line 191, in get_policy
    policy = jwt.decode(token, verify=False).get('x-ms-policy', '')
  File "/opt/az/lib/python3.6/site-packages/jwt/api_jwt.py", line 119, in decode
    decoded = self.decode_complete(jwt, key, algorithms, options, **kwargs)
  File "/opt/az/lib/python3.6/site-packages/jwt/api_jwt.py", line 87, in decode_complete
    'It is required that you pass in a value for the "algorithms" argument when calling decode().'
jwt.exceptions.DecodeError: It is required that you pass in a value for the "algorithms" argument when calling decode().

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- `az attestation policy show --resource-group {} --name {} --attestation-type {}`

## Environment Summary
```
Linux-4.15.0-96-generic-x86_64-with-debian-buster-sid
Python 3.6.10
Installer: DEB

azure-cli 2.27.2

Extensions:
attestation 0.2.0

```

### PR DESCRIPTION
**Pull request by YuanyuanNi**
Fixes: https://github.com/Azure/azure-cli-extensions/issues/3937
Fixes: https://github.com/Azure/azure-cli-extensions/issues/4120

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
attestation

File: src/attestation/azext_attestation/manual/custom.py [hunk 1]
```diff
@@ -188,7 +188,7 @@ def get_policy(cmd, client, attestation_type, resource_group_name=None, provider
 
     if token:
         import jwt
-        policy = jwt.decode(token, verify=False).get('x-ms-policy', '')
+        policy = jwt.decode(token, algorithms=['RS256'], options={"verify_signature": False}).get('x-ms-policy', '')
```
Comment (jsntcy) [inline src/attestation/azext_attestation/manual/custom.py:191]: There should be some other places using "jwt.decode", please update them as well.

**Comment by jsntcy** [comment]
Close as it's out of date and covered by https://github.com/Azure/azure-cli-extensions/pull/5081

**Pull request closed without merging**